starfish/pipeline/filter/gaussian_high_pass.py

```diff
@@ -68,9 +77,9 @@ def filter(self, stack: Stack) -> None:
 
         """
 
-        high_pass: Callable = partial(self.gaussian_high_pass, sigma=self.sigma)
-        stack.image.apply(high_pass)
+        high_pass: Callable = partial(self.high_pass, sigma=self.sigma)
```

Comment:
Idea: verify that if self.sigma is a tuple, that the length is equal to the shape of the image.

Reply:
Done.